### ISSUE TITLE
Alt tab only switching windows the first time

### ISSUE THREAD

**Issue by ZVNexus** [pop-os/cosmic-comp#538]
If you have two windows open and try to alt tab between them, it works the first time but every attempt after results in the alt tab launcher showing up but it just remains in the same window. Have to click tab again to actually get it to switch windows.

**Comment by senntore** [comment]
I don't think it is switching to any window at all. It is selecting the window in the order it is listed. Normally it should select the next window up first use but that is not what is happening here.

**Comment by git-f0x** [comment]
It's planned to improve the behavior with alpha 2 (there's an issue in cosmic-launcher for recency-based Alt+Tab cycling).

**Comment by WatchMkr** [comment]
Recency based alt+tab has been implemented.
